nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -83,10 +83,12 @@ Costmap2DROS::Costmap2DROS(const std::string & name, tf2_ros::Buffer & tf)
   set_parameter_if_not_set("origin_y", 0.0);
   set_parameter_if_not_set("footprint", "[]");  
   set_parameter_if_not_set("footprint_padding", 0.01);
-  set_parameter_if_not_set("robot_radius", 0.1); 
+  set_parameter_if_not_set("robot_radius", 0.46);
```

Comment:
Why? 0.1 is about right for the turtlebot.

Reply:
The default value in the original cfg file had the robot radius at 0.46, so I thought that I had mistakenly given it a different default value before. I can change it back, though